DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -351,18 +351,6 @@ public void RemoveDelegateUser(int delegateId)
             using var transaction = new TransactionScope();
             try
             {
-                var existingSessions = connection.Query<int>(
-                    @"SELECT SessionID FROM Sessions WHERE CandidateID = @delegateId",
-                    new { delegateId }
-                );
-
-                if (existingSessions.Any())
-                {
-                    throw new UserAccountInvalidStateException(
-                        $"Delegate user id {delegateId} cannot be removed as they have already started a session."
-                        );
-                }
-

```

Comment:
Ah - should we Dispose in the finally block?

Reply:
The dispose is handled in the catch block - dispose reverts the transaction so we don't want to fire it of there's no exception.